### PR TITLE
Automatically increase stack size

### PR DESCRIPTION
**Pull request by JasonGross**
Requested at https://coq.zulipchat.com/#narrow/stream/247791-fiat-crypto/topic/Fiat.20crypto.20and.20flambda/near/303701926

### CODE REVIEW

File: Makefile.standalone [hunk 1]
```diff
@@ -26,17 +26,23 @@ endif
 	standalone-haskell install-standalone-haskell uninstall-standalone-haskell \
 	#
 
+ifneq ($(SET_ULIMIT_STACK),1)
+CHECK_STACK_LIMIT:=etc/check_stack_limit.sh || true
+else
+CHECK_STACK_LIMIT:=etc/check_stack_limit.sh 2>/dev/null || ulimit -S -s 32768 || etc/check_stack_limit.sh || true
```
Comment (andres-erbsen) [inline Makefile.standalone:32]: rename to `STACK_LIMIT`?

maybe pass the value of `SET_ULIMIT_STACK` to ulimit?

Comment (JasonGross) [reply Makefile.standalone:32]: > maybe pass the value of `SET_ULIMIT_STACK` to ulimit?

That sort-of defeats the purpose of users not having to think about what stack value they need to pass

Comment (andres-erbsen) [reply Makefile.standalone:32]: if we don't want to allow configurability, perhaps we should do this unconditionally?

Comment (JasonGross) [reply Makefile.standalone]: Good idea.  I've updated it, does it look good now?

**Pull request merged**